lrlex/src/lib/builder.rs

```diff
@@ -113,8 +113,8 @@ where
         self.process_file(inp, outp)
     }
 
-    /// Statically compile the `.l` file `inp` into Rust, placing the output into `outp`. The
-    /// latter defines a module with a function `lexerdef()`, which returns a
+    /// Statically compile the `.l` file `inp` into Rust, placing the output to the file `outp`.
```

Comment:
Needs to be "into" the file.

Reply:
This one is yet to be fixed?

Reply:
I'm an idiot. I changed it but didn't save the file before comitting.

Reply:
Fixed.